src/components/icons/CameraIssue.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../helpers';
+
+/**
+ * Camera issue SVG Icon
+ *
+ * @param {object} props  Data passed to the component
+ * @param {number} props.size  Width and height of the icon
```

Comment:
this icon does not seem to be square, so a single "size" prop does not make sense.

You can just leave the size (96x88) hardcoded and remove the size prop.